presqt/api_v1/serializers/resource.py

```diff
@@ -11,16 +13,55 @@ class ResourcesSerializer(serializers.Serializer):
     id = serializers.CharField(max_length=256)
     container = serializers.CharField(max_length=256)
     title = serializers.CharField(max_length=256)
-    detail = serializers.SerializerMethodField()
+    links = serializers.SerializerMethodField()
+
+    def get_links(self, instance):
+        """
+        Translate the `links` property to a custom array of Hyperlink values.
+
+        Parameters
+        ----------
+        instance : Target Obj instance
+
+        Returns
+        -------
+        Hyperlink url for Target detail API endpoint
+        """
+        links = []
+
+        resource_detail, resource_download, resource_upload = target_checker(
```

Comment:
You may want to think about returning something smaller like a list...maybe a list of actions that are supported. And then instead of checking if that action is true just check if it's in the list.